features/networking/operator.feature

```diff
@@ -14,4 +14,4 @@ Feature: Operator related networking scenarios
 
     Given evaluation of `cluster_operator('network').versions` is stored in the :operator_version clipboard
     #Confirm whether network operator version matches with ocp version`
-    Then the expression should be true> cb.operator_version="<%= cb.ocp_version%>"
+    Then the expression should be true> cb.operator_version=cb.ocp_version
```

Comment:
@anuragthehatter I think this is incorrect...assignment will be true....I think you meant to do a comparsion which should be `==`

Reply:
Agree, @pruan-rht. My bad i have used '==' in another epression above this but seems like a typo in this 2nd expression. Good catch. Thanks!